tests/www/views/test_views_base.py

```diff
@@ -35,16 +34,14 @@ def test_index(admin_client):
     check_content_in_response('DAGs', resp)
 
 
-def test_doc_urls(admin_client):
+def test_doc_urls(admin_client, monkeypatch):
+    # Mocking this way is tying the test closer to the implementation much more than I'd like. :shrug:
+    from airflow.www.views import AirflowBaseView
+
+    monkeypatch.setitem(AirflowBaseView.extra_args, 'get_docs_url', lambda _: "!!DOCS_URL!!")
     resp = admin_client.get('/', follow_redirects=True)
-    if "dev" in version.version:
```

Comment:
I did this, as otherwise we are just re-implementing the get_docs_url logic here in the test.